src/main/java/uk/gov/justice/probation/courtcasematcher/model/MessageAttributes.java

```diff
@@ -1,16 +1,25 @@
 package uk.gov.justice.probation.courtcasematcher.model;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.AccessLevel;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import uk.gov.justice.probation.courtcasematcher.messaging.HearingEventType;
 import uk.gov.justice.probation.courtcasematcher.messaging.model.MessageType;
 
+import java.io.Serializable;
+
 @Data
 @AllArgsConstructor
 @NoArgsConstructor(access = AccessLevel.PRIVATE, force = true)
-public class MessageAttributes {
+public class MessageAttributes  implements Serializable {
+    @JsonProperty("messageType")
     private final MessageType messageType;
+
+    @JsonProperty("hearingEventType")
     private final HearingEventType hearingEventType;
+
+    @JsonProperty("ExtendedPayloadSize")
```

Comment:
How annoying that this has an uppercase `E` at the start
The other two JsonProperty annotations seem unnecessary as the keys already match case
I am also curious as to why serializable has been implemented